network-api/networkapi/templates/foundation-base.html

```diff
@@ -72,6 +72,34 @@
         </div>
       </div>
     </div>
+    {% if showHeroBanner %}
+    <div id="hero">
+      <div class="container">
+        <div class="row">
+          <div class="col-md-12 col-lg-6 py-5">
+            <div class="mb-4">
+              <h1 class="h1-white">Fellowships</h1>
+            </div>
+            <p class="lead-black">Our Fellowships empower bright minds in the fields of public policy, science and journalism.</p>
+          </div>
+        </div>
+      </div>
+    </div>
+    {% endif %}
+    <div id="fellowships-nav" class="mb-5">
```

Comment:
Is the nav supposed to be available to all pages extending foundation-base.html?

Reply:
From what I see on https://projects.invisionapp.com/d/main#/console/10016110/263404839/preview ... yea I think the fellowship nav is always there.

It seems like the homepage is the only page that has the hero banner though. Did I understand this correctly, @taisdesouzalessa ?